hack/arktos-cni.rc

```diff
@@ -33,7 +33,7 @@ setup_cni_conf() {
 
     echo "copying ${conf_source_file} to ${conf_target_file}..."
     local cni_conf_dir=$(dirname ${conf_target_file})
-    local backup_dir="$(dirname ${cni_conf_dir})/$(basename ${cni_conf_dir})_"$(date -d "today" +"%Y%m%d%H%M")
+    local backup_dir="/var/backups/$(basename ${cni_conf_dir})_"$(date -d "today" +"%Y%m%d%H%M")
```

Comment:
This might have compatibility issues with other cni plugin

Reply:
Test with Mizar and default arktos-up both tested. Revert arktos-up changes for mizar. It will be fully tested and merged in a different PR.